docs/sources/mimir/operators-guide/get-started/_index.md

```diff
@@ -9,24 +9,26 @@ weight: 10
 
 # Get started with Grafana Mimir
 
-There are two different options for getting started with Grafana Mimir:
+There are two different ways that you to get started with Grafana Mimir:
 
-- The written tutorial provides a series of imperative commands to start a single Mimir process.
-- The visual tutorial (in the form of a video) uses `docker-compose` to declaratively deploy multiple Mimir processes.
+- The written instructions that follow help you start a single Mimir process by using [_imperative_ Kubernetes commands](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/imperative-command/).<p>
```

Comment:
This documentation page has nothing to do with Kubernetes, so I don't think it's correct what you're writing here ("using imperative Kubernets commands").

Reply:
What do the terms relate to?

Reply:
I have removed the terms for now, and can add them back in if we define them.